run/helm/snap/templates/daemonset.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: snap
+  namespace: {{ .Values.namespace }}
+  labels:
+    kubernetes.io/cluster-service: "true"
```

Comment:
This is used for [Kubernetes cluster add-ons](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons).
In the initial version of kubesnap, we were installing snap as an add-on. Kubernetes just makes sure that add-on is always running:
> If an add-on is deleted, it will be recreated automatically.
> If an add-on is updated through Apiserver, it will be reconfigured to the state given by the supplied fields in the initial config.